src/lib/__tests__/featureChecking-test.ts

```diff
@@ -0,0 +1,31 @@
+import { getFeatureConfig, isFeatureEnabled } from '../featureChecking';
+
+const features = {
+    shouldUseCar: {
+        enabled: true,
+    },
+    shouldUsePlane: {
+        enabled: false,
+    },
+};
+
+describe('isFeatureEnabled', () => {
```

Comment:
Minor, but we typically maintain one top-level `describe` per test file. Can we wrap these two?